webpack.config.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+
+const CleanWebpackPlugin  = require('clean-webpack-plugin'),
+  HtmlWebPackPlugin       = require('html-webpack-plugin'),
+  MiniCssExtractPlugin    = require('mini-css-extract-plugin'),
+  OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin'),
+  Package                 = require('./package.json'),
+  path                    = require('path'),
+  project_config          = require('./config.js'),
+  UglifyJsPlugin          = require('uglifyjs-webpack-plugin');
+
+const versionJS        = Package.webpack_bundle_version_js,
+      versionCSS       = Package.webpack_bundle_version_css,
+      buildDestination = './dist';
+
+// project_config overwrites default_config, edit project_config to customize the values below or add new fields
+const default_config = {
```

Comment:
This could just be an empty object, no?  Do we even need these defaults? They are set in the config file already